src/main/scala/org/clulab/concepts/ConceptDiscoverer.scala

```diff
@@ -113,10 +118,14 @@ class ConceptDiscoverer(
   def loadConcepts(file_loc: String): Vector[Concept]={
     val conceptLocations = mutable.Map.empty[String, Set[DocumentLocation]]
       .withDefaultValue(Set.empty)
-    scala.io.Source.fromFile(file_loc).getLines().foreach{line =>
-      val phrase = line.stripLineEnd.split('\t')(0)
-      val locations = line.stripLineEnd.split('\t')(1)
-      conceptLocations(phrase) += DocumentLocation(locations.split(' ')(0), locations.split(' ')(1).toInt)
+
+    Sourcer.sourceFromFile(new File(file_loc)).autoClose { source =>
+      val lines = source.getLines().toArray
+      lines.foreach { line =>
+        val phrase = line.stripLineEnd.split('\t')(0)
```

Comment:
There's a way to do something like
```
Array(phrase, locations) = line.stripLineEnd.split('\t')
```
so that the split only need happen once.  Something like it might work below for locations.split().

Reply:
do you want me to make that change?

Reply:
Optionally, or eventually, and only if you want cool code.